.github/workflows/localization_branch_sync.yml

```diff
@@ -0,0 +1,35 @@
+name: Sync Localization Branch
+on:
+  pull_request:
+    branches:
+      - 'main'
+    types: [closed]
+
+concurrency: 
+  group: 'Localization-Sync'
+  cancel-in-progress: false
+
+jobs:
+  replaceLocalizationBranch:
+    name: 'Replace Localization Branch'
+    runs-on: ubuntu-latest
+    steps:
+    - name: 'Check for Fork'
+      run: echo "Action cannot be executed on forks"
```

Comment:
Sorry I forgot to mention, do an exit 1 to stop the execution of the other steps.